armi/nuclearDataIO/cccc/tests/test_gamiso.py

```diff
@@ -31,6 +31,12 @@ def setUp(self):
         self.xsLib = xsLibraries.IsotxsLibrary()
 
     def test_compare(self):
+        """Compare the input binary GAMISO file.
+
+        .. test:: Test reading GAMISO files.
+            :id: T_ARMI_NUCDATA_GAMISO
+            :tests: R_ARMI_NUCDATA_GAMISO
+        """
```

Comment:
technically the requirement states read and write. This only reads. Do we need to write a test to write too?

Reply:
Deal. Adding a test now.

Reply:
Done.